pkg/graveler/ref/db_manager.go

```diff
@@ -502,3 +526,64 @@ func (m *DBManager) addGenerationToNodes(nodes map[graveler.CommitID]*CommitNode
 		nodesCommitIDs = nextIterationNodes
 	}
 }
+
+// rWorker writes a repository, and all repository related entities, to fd
+func rWorker(rChan <-chan *graveler.RepositoryRecord) error {
```

Comment:
The worker does so little that it's almost not worth parallelizing on it, unless this is a preparation for other entities under the worker like branches, etc.